File: Example/Objc/Example/ViewController.m
//
//  ViewController.m
//  Example
//
//  Created by David Cordero on 13.06.18.
//  Copyright © 2018 David Cordero. All rights reserved.
//

#import "ViewController.h"

@import FocusTvButton;

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    FocusTvButton *button = [[FocusTvButton alloc] initWithFrame:CGRectMake(100, 100, 200, 200)];
    button.focusedBackgroundColor = UIColor.greenColor;
    button.normalBackgroundColor = UIColor.lightGrayColor;
    button.cornerRadius = 0.5 * button.bounds.size.width;
    [button setTitle:@"Button" forState:UIControlStateNormal];

    [self.view addSubview:button];
}

@end

File: Example/Example/ViewController.swift
//
//  ViewController.swift
//  Example
//
//  Created by David Cordero on 02/09/16.
//  Copyright © 2016 David Cordero. All rights reserved.
//

import UIKit
import FocusTvButton

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        let button = FocusTvButton(frame: CGRect(x: 100, y: 100, width: 200, height: 200))
        button.focusedBackgroundColor = .green
        button.normalBackgroundColor = .lightGray
        button.cornerRadius = button.bounds.width / 2
        button.setTitle("Button", for: .normal)
        
        view.addSubview(button)
    }
}
